watertap/data/techno_economic/metab.yaml

```diff
@@ -0,0 +1,139 @@
+default:
```

Comment:
minor comment but filename should be metab_zo.yaml to be consistent with all other units

Reply:
Nevermind- mixed up with unit model naming.